packages/wrangler/src/pages/build-env.ts

```diff
@@ -9,17 +12,37 @@ import type {
 export type PagesBuildEnvArgs = StrictYargsOptionsToInterface<typeof Options>;
 
 export function Options(yargs: CommonYargsArgv) {
-	return yargs;
+	return yargs.positional("projectDir", {
+		type: "string",
+		description: "The location of the Pages project",
+	});
 }
 
 export const Handler = async (args: PagesBuildEnvArgs) => {
-	const config = readConfig(undefined, {
+	if (!args.projectDir) {
```

Comment:
what's the reason for making this mandatory. Would it make sense to default to the current working directory?

Reply:
Potentially, but given this is a hidden command only meant to be used in CI (where the argument will always be set), I'm not sure if we should?

Reply:
that would have been my next suggestion, to make it hidden if it's only needed in CI 😅 . I didn't notice it already was. Then totes fine the way it is now